.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI build
+on:
+  pull_request:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```

Comment:
I'm fine bumping the project to use 1.14 or even 1.15, but we'll need to update `go.mod` and the [Prow presubmits](https://github.com/kubernetes/test-infra/blob/f34ec251f8b943950bbd0718e13c47b661ea97da/config/jobs/kubernetes-sigs/aws-encryption-provider/presubmits.yaml) container images

Reply:
Sure PR for prow jobs: https://github.com/kubernetes/test-infra/pull/20702